src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1484,6 +1495,9 @@ struct __ServiceCallContext {
 
     // Authentication info
     PAuthInfo pAuthInfo;
+
+    // -------- V1 compat --------

```

Comment:
This should be V0 compat, marking the end of V0, to match the format in our other versioned structs.  See for example `ClientInfo` and `StreamCaps`